### DISCUSSION TITLE
استفاده از حساب داینامیک بانک ملت و متد AddMellatCumulativeAccounts

### DISCUSSION THREAD

**Discussion by ShahbaziMohammad** [Sina-Soltani/Parbad#229] [Q&A] [answered]
با سلام
می خواهم از پرداخت تجمیعی بانک ملت استفاده کنم و طبق مستندات و راهنمایی صورت گرفته مقادیر سرویس آی دی و مبلغ را تنظیم می کنم ولی موقع فراخوانی متد
 RequestAsync 
خطای 
Object reference not set to an instance of an object
را می گیرم . دوستان کسی می تونه راهنمایی کنه؟ 

نمونه کد استفاده شده :

                var result = await _onlinePayment.RequestAsync(invoice =>
                {
                    var inv = invoice
                        .SetTrackingNumber(trackingNumber)
                        .SetAmount(50000)
                        .SetCallbackUrl(paymentBackUrl)
                        .AddMellatCumulativeAccount(1,20000)
                        .AddMellatCumulativeAccount(2,30000)
                        .UseMellat();
                });

پاسخی که دریافت می کنم:

{"status":1,"gatewayTransporter":{"descriptor":null},"isSucceed":false,"trackingNumber":2021072706290012265,"amount":{"value":50000.0},"gatewayName":"Mellat","gatewayAccountName":null,"message":"Object reference not set to an instance of an object.","additionalData":{},"databaseAdditionalData":{}}

با تشکر

**Comment by Sina-Soltani** [comment, marked as answer]
درود

بررسی میکنم

**Reply by Sina-Soltani** [reply]
مشکل برطرف شد
لطفا نسخه جدید رو نصب کنید
`Install-Package Parbad -Version 3.7.1-beta2`

**Reply by ShahbaziMohammad** [reply]
سلام 
مشکل بر طرف شد
بسیار سپاسگزام
